vars/handlePipelineStepErrors.groovy

```diff
@@ -10,10 +10,10 @@ void call(Map parameters = [:], body) {
     def message = ''
     try {
         if (stepParameters == null && stepName == null)
-            error "step handlePipelineStepErrors requires following mandatory parameters: stepParameters, stepName"
+            error "The step handlePipelineStepErrors requires following mandatory parameters: stepParameters, stepName"
```

Comment:
[nit] not covered by the scope of the pull request `:-)    ->    ;-)    ->     :-)`

Reply:
Our users deserve a whole sentence, don't they? :)

Reply:
Sure. Nevertheless scope of PR is "Don't print messages all in uppercase letters". The "The" is unrelated. Never mind. Was labeled with `[nit]` and decorated with some smilies. Basically kidding ...